spec/controllers/embed_controller_spec.rb

```diff
@@ -27,5 +28,11 @@
       json = JSON.parse(response.body)
       expect(json["html"]).to be_blank
     end
+
+    it 'passes along configured additional keys' do
+      allow(OEmbed::Providers).to receive(:get).with('http://some/uri', canvas_index: '5').and_return oembed_obj
+      get :show, params: { url: 'http://some/uri', canvas_index: '5' }
+      expect(response.status).to eq 200
```

Comment:
you can use `expect(response).to have_http_status(:ok)` here if you prefer

Reply:
Yep! I  was just following the pattern in the tests above for consistency.